ui/src/components/TestRunner.tsx

```diff
@@ -14,12 +14,14 @@ import { socket } from "helpers/socketConnection";
 
 const DROPDOWN_SEPS = [6, 10, 12, 24, 31];
 
+const CONFIG_SEPS = [6, 12, 31];
```

Comment:
ideally this would be configured somewhere in @stellar/anchor-tests and just imported here